oak_containers/proto/interfaces.proto

```diff
@@ -47,6 +52,10 @@ service Launcher {
   // application config. The orchestrator will later, separately expose this
   // config to the application.
   rpc GetApplicationConfig(google.protobuf.Empty) returns (GetApplicationConfigResponse) {}
+
+  // Sends Attestation Evidence containing the Attestation Report with corresponding measurements
+  // and public keys to the Launcher.
```

Comment:
This is expected to be called exactly once, at the end of the DICE chain, right?

Reply:
Correct, updated the comment